packages/design/package.json

```diff
@@ -26,20 +26,22 @@
     "build:removeTempFiles": "rm -f src/foundation.js && mv foundation.d.ts.temp foundation.d.ts && mv foundation.js.temp foundation.js",
     "build:iconCssStyle": "node buildIconStyles.js && npm run build:iconCssStyleTypes",
     "build:iconCssStyleTypes": "tsc --declaration --emitDeclarationOnly --allowJs src/icons/iconStyles.ts --outDir ./src/icons",
-    "clean": "rm -rf dist/* icons/* colors.js foundation.css iconStyles.* tsconfig.tsbuildinfo",
-    "prepare": "npm run clean; npm run build",
+    "clean": "rm -rf dist/* icons/* colors.js foundation.* iconStyles.* tsconfig.tsbuildinfo ./src/icons/iconStyles.*",
+    "prepare": "npm run clean && npm run build",
+    "postinstall": "npm run prepare",
     "ci:typescript-compile": "npm run build:ts"
   },
-  "dependencies": {
-    "classnames": "^2.2.6"
+  "optionalDependencies": {
+    "@jobber/fonts": "^1.0.0-pre-release.0"
   },
-  "devDependencies": {
-    "@types/classnames": "^2.2.10",
+  "dependencies": {
```

Comment:
Needed to move the postcss stuff to a dependency because the package needs to build when installed so that the fonts can be bundled "automagically"